src/AspNet.Security.OAuth.JumpCloud/JumpCloudAuthenticationExtensions.cs

```diff
@@ -68,7 +68,7 @@ public static AuthenticationBuilder AddJumpCloud([NotNull] this AuthenticationBu
     public static AuthenticationBuilder AddJumpCloud(
         [NotNull] this AuthenticationBuilder builder,
         [NotNull] string scheme,
-         string? caption,
+        [CanBeNull] string caption,
```

Comment:
If it can be null it should still be nullable (`string?`) too.

Reply:
This doesn't accept nullable types:
![image](https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/assets/9937803/2e2ea520-2f2a-43fd-b63a-e977d0cb2763)

I have followed the Okta example for this

Reply:
OK - I thought we'd turned them on, but I guess not!